tests/unit/InitialVelocityTests.swift

```diff
@@ -44,75 +44,72 @@ class InitialVelocityTests: XCTestCase {
   }
 
   func testVelocityAmplitudeMatchesDisplacementWithPositiveDisplacement() {
-    let timing = MotionTiming(delay: 0,
-                              duration: 0.7,
-                              curve: .init(type: .spring, data: (1, 1, 1, 50)),
-                              repetition: .init(type: .none, amount: 0, autoreverses: false))
-    animator.animate(with: timing, to: CALayer(), withValues: [0, 100], keyPath: .opacity)
+    animate(from: 0, to: 100, withVelocity: 50)
 
-    XCTAssertEqual(addedAnimations.count, 1)
-    let animation = addedAnimations.first as! CASpringAnimation
-    XCTAssertEqual(animation.initialVelocity, 0.5)
+    XCTAssertEqual(addedAnimations.count, 3)
+    addedAnimations.flatMap { $0 as? CASpringAnimation }.forEach { animation in
+      XCTAssertEqual(animation.initialVelocity, 0.5)
```

Comment:
Now that we're testing 3 different things in-line, I'd prefer to see a more descriptive failure message that calls out the start/end values (including types). It would make failures easier to diagnose from logs.

Reply:
Done.

Reply:
Example failures:

```
InitialVelocityTests.swift: test failure: InitialVelocityTests.testVelocityAmplitudeMatchesDisplacementWithPositiveDisplacement() failed: XCTAssertEqual failed: ("0.5") is not equal to ("0.3") - from: -100, to: 0, withVelocity: 50.0

InitialVelocityTests.swift: test failure: InitialVelocityTests.testVelocityAmplitudeMatchesDisplacementWithPositiveDisplacement() failed: XCTAssertEqual failed: ("0.5") is not equal to ("0.3") - from: NSPoint: {-100, -100}, to: NSPoint: {0, 0}, withVelocity: 50.0

InitialVelocityTests.swift: test failure: InitialVelocityTests.testVelocityAmplitudeMatchesDisplacementWithPositiveDisplacement() failed: XCTAssertEqual failed: ("0.5") is not equal to ("0.3") - from: NSSize: {-100, -100}, to: NSSize: {0, 0}, withVelocity: 50.0
```

Reply:
Awesome!